helm/grafana-agent/templates/logging-credentials-secret.yaml

```diff
@@ -0,0 +1,14 @@
+{{- if .Values.loggingSecret.enabled }}
```

Comment:
I like the idea of an extra secret but do not like the custom secret parameters. I would have liked to see a more generic approach that could have been proposed upstream as well like https://github.com/prometheus-community/helm-charts/blob/main/charts/kube-prometheus-stack/templates/prometheus/extrasecret.yaml for the secret and this to read the secret data https://grafana.com/docs/agent/latest/flow/reference/components/remote.kubernetes.secret/#example ?

Reply:
I like your idea :-)